### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by dafinoer**
hii, i'm doing some update in readme about navigate to other screen using name.

### CODE REVIEW

**Review by Nipodemos — APPROVED** [review]
great!

**Review by eduardoflorence — COMMENTED** [review]
Named routes require additional configuration, so it would be important to add the comment below:

File: README.md [hunk 1]
```diff
@@ -226,6 +226,13 @@ Navigate to new screen:
 Get.to(NextScreen());
 ```
 
+Navigate to new screen with name:
```
[the remark this answers is not in the document]
Comment (Nipodemos) [reply README.md:229]: that's actually a good idea, we need more links that lead to the complete docs

**Comment by jonataslaw** [comment]
Now LGTM. 
Thanks for your contribution!

**Pull request merged**